### ISSUE TITLE
Docs feedback: /operators-guide/configure/configure-object-storage-backend.md

### ISSUE THREAD

**Issue by dggmsa** [grafana/mimir#4224]
the example as set with:
```
common:
  storage:
    backend: s3
    s3:
      endpoint: s3.us-east-2.amazonaws.com
      region: us-east
      secret_access_key: "${AWS_SECRET_ACCESS_KEY}" # This is a secret injected via an environment variable
      access_key_id: "${AWS_ACCESS_KEY_ID}" # This is a secret injected via an environment variable
```
This can be change to:

```
common:
  storage:
    backend: s3
    s3:
      endpoint: s3.us-east-2.amazonaws.com
      region: us-east-2
      secret_access_key: "${AWS_SECRET_ACCESS_KEY}" # This is a secret injected via an environment variable
      access_key_id: "${AWS_ACCESS_KEY_ID}" # This is a secret injected via an environment variable
```
the region looks confuse whitout the number of region 

region: us-east-2

**Comment by pracucci** [comment]
Could you open a PR fixing it, please?

**Comment by dggmsa** [comment]
Yes, https://github.com/grafana/mimir/pull/4225

**Comment by pracucci** [comment]
Closed by #4225. Thanks! ❤️
